src/main/java/net/syscon/elite/api/resource/OffenderMovementsResource.java

```diff
@@ -110,4 +111,13 @@ ScheduledPrisonToPrisonMove prisonToPrison(@ApiParam(value = "The offender booki
     CourtHearing courtHearingDateAmendment(@ApiParam(value = "The offender booking to associate the update with.", required = true) @PathVariable("bookingId") Long bookingId,
                                            @ApiParam(value = "The  court hearing to be updated.", required = true) @PathVariable Long hearingId,
                                            @ApiParam(value = "The amendments for the scheduled court hearing.", required = true) @RequestBody @Valid CourtHearingDateAmendment amendment);
+
+    @DeleteMapping("/{bookingId}/court-hearings/{hearingId}/cancel")
+    @ApiOperation(value = "Cancels the scheduled court hearing an offender.", notes = "Cancels the scheduled court hearing an offender.", nickname = "cancelCourtHearing")
```

Comment:
Does this sentence make sense? `Cancels the scheduled court hearing an offender`?

Reply:
No sir it does not.  Will fix.